source/transactions/tests/bulk.yml

```diff
@@ -70,19 +70,19 @@ tests:
             - name: deleteOne
               arguments:
                 filter: {_id: 4}
-            - name: deleteMany
-              arguments:
-                filter: {_id: {$gte: 6}}
             - name: updateMany
               arguments:
                 filter: {_id: {$gte: 2}}
                 update: {$set: {z: 1}}
+            - name: deleteMany
```

Comment:
Can you add a comment explaining why deleteMany needs to separated from deleteOne (similar to the replaceOne comment above)?